src/Adapters/Files/GridFSBucketAdapter.js

```diff
@@ -14,17 +14,21 @@ import defaults from '../../defaults';
 export class GridFSBucketAdapter extends FilesAdapter {
   _databaseURI: string;
   _connectionPromise: Promise<Db>;
+  _mongoOptions: Object;
 
-  constructor(mongoDatabaseURI = defaults.DefaultMongoURI) {
+  constructor(mongoDatabaseURI = defaults.DefaultMongoURI, mongoOptions = {}) {
     super();
     this._databaseURI = mongoDatabaseURI;
+    this._mongoOptions = mongoOptions;
+    this._mongoOptions.useNewUrlParser = true;
```

Comment:
does it not work with the old parser?

if it doesn't, this looks fine.

if the old parser does work, then something like:

```
const defaultMongoOptions = { useNewUrlParser: true };
this._mongoOptions = Object.assign(defaultMongoOptions, mongoOptions);
```

Reply:
I don't know as I use AWS S3. https://github.com/parse-community/parse-server/issues/5525 posted that this change fixed it for them. Should I make your requested change?

Reply:
well, not without testing it, but I think so, yes. I don't see any reason not to allow a user to override the default.

Reply:
Oh I see, this will allow for both.